uktrade/data-hub-frontend
# Add my pipeline placeholder

## Description of change

Add a new "My pipeline" tab to the dashboard with sub tabs. This re-uses the existing `TabNav` component and some tweaks were required to get everything working correctly. 

This is the final PR to get merged for this feature and there will be some subsequent PRs coming from @adamledwards and @debitan that will merge into this PR before it will all be merged to release the feature.

## Test instructions

Go to the dashboard and you should see a "My pipleline" tab, this should show three sub tabs and each one should show some different content - all should be working with React Router as well.

## Screenshots
### Before

<img width="975" alt="Screenshot 2020-05-15 at 15 46 06" src="https://user-images.githubusercontent.com/1481883/82063353-45ad5d00-96c3-11ea-97d4-9221160474e9.png">

### After

<img width="976" alt="Screenshot 2020-05-15 at 15 45 34" src="https://user-images.githubusercontent.com/1481883/82063383-4f36c500-96c3-11ea-8363-aca417462069.png">


## Checklist

[//]: # "When submitting a PR make sure the code review guidelines have been satisfied.
https://github.com/uktrade/data-hub-frontend/blob/master/docs/Code%20review%20guidelines.md"
